lib/pf9-builder/template/hosts-global.tpl

```diff
@@ -0,0 +1,9 @@
+[all]
+[all:vars]
+#ansible_ssh_pass=Pl@tform9
+#ansible_sudo_pass=Pl@tform9
+ansible_ssh_private_key_file=~/cm-master
+manage_network=True
+live_migration=True
+nested_virt=True
+kernel_same_page_merging=False
```

Comment:
**these variables should be removed:**
live_migration=True
nested_virt=True
kernel_same_page_merging=False

we should set this to ansible_ssh_pass / sudo_pass to winterwonderland to align with the main hosts doc

ssh key should also be more generic vs. yours